app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -19,3 +19,11 @@ CONFIG_PCM_CONVERTER_FORMAT_S24_C32_AND_S24_C24=y
 CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S16_C32=y
 CONFIG_PM_POLICY_CUSTOM=y
 CONFIG_L3_HEAP=y
+CONFIG_LOG=y
+CONFIG_LOG_MODE_DEFERRED=y
+CONFIG_LOG_BACKEND_ADSP=n
+CONFIG_LOG_BACKEND_ADSP_HDA=y
+CONFIG_LOG_BACKEND_ADSP_HDA_SIZE=8192
+CONFIG_LOG_BACKEND_ADSP_HDA_FLUSH_TIME=100
+CONFIG_LOG_FUNC_NAME_PREFIX_INF=y
```

Comment:
This looks like it was squashed from totally not related commits. Maybe better to move this to commit where this file was added?